### PR TITLE
Added label for remote yoyo code on captive portal and removed text box when none updatable. Also added extra logic to ensure captive portal is in correct setup mode 

### PR DESCRIPTION
**Pull request by andysheen**

### CODE REVIEW

File: data/index.html [hunk 1]
```diff
@@ -20,10 +20,12 @@
...
-          <p>Your Yo-Yo Code is:</p>
+          <p>Your Yo-Yo Code:</p>
           <p id="local-scad-code">XXXX-XXXXX</p>
+          <p>Remote Yo-Yo Code:</p>
+          <p id ="remote-scad-code">XXXX-XXXXX</p>
         </div>
-    
+        
         <div class="mb-4" id="remoteMacForm">
           <input id="remote-scad-code-input" class="form-control" inputmode="numeric" pattern="[0-9 ]+" type="text" placeholder="Remote Yo-Yo Code">
```
Comment (mikevanis) [inline data/index.html:30]: Add label like this just above the input tag:
```
<label for="remote-scad-code-input">Remote Yo-Yo Code</label>
```

File: data/index.html [hunk 2]
```diff
@@ -20,10 +20,12 @@
...
         <div class="lead" id="macForm">
-          <p>Your Yo-Yo Code is:</p>
+          <p>Your Yo-Yo Code:</p>
           <p id="local-scad-code">XXXX-XXXXX</p>
+          <p>Remote Yo-Yo Code:</p>
+          <p id ="remote-scad-code">XXXX-XXXXX</p>
         </div>
-    
+        
         <div class="mb-4" id="remoteMacForm">
```
Comment (mikevanis) [inline data/index.html:29]: This should also have a class of form-group.
```
<div class="mb-4 form-group" id="remoteMacForm">
```

File: data/script.js [hunk 3]
```diff
@@ -16,8 +16,13 @@ function configure(json) {
...
     $('#local_ssid').text(json.local_ssid);
     //TODO: use the password length to display
-    if(json.remote_mac != "") $('#remote-scad-code-input').attr('value', formatScadCode(json.remote_mac));
     $('#local-scad-code').text(formatScadCode(json.local_mac));
+    if(json.remote_mac != ""){
+        $('#remote-scad-code').text(formatScadCode(json.remote_mac));
+    } else {
+        $('#remote-scad-code').hide();
+    }
+
```
Comment (mikevanis) [inline data/script.js]: This should populate the span inside the #remote-scad-text p, and hide the input. If there is no remote_mac, it should hide the #remote-scad-text p.
```
if (json.remote_mac != "") {
    $('#remote-scad-code').text(formatScadCode(json.remote_mac));
    $('#remote-scad-code-input').hide();
} else {
    $('#remote-scad-text').hide();
}

**Review by mikevanis — CHANGES_REQUESTED** [review]
Some suggestions for making it look semantically consistent across all scenarios. My suggestion is that the remote yoyo code should be displayed in the input box and it should be read only.

File: data/index.html [hunk 4]
```diff
@@ -20,10 +20,12 @@
       <img src="logo_yoyo.png" alt="" width="200" height="96">
       <div id="config">
         <div class="lead" id="macForm">
-          <p>Your Yo-Yo Code is:</p>
+          <p>Your Yo-Yo Code:</p>
           <p id="local-scad-code">XXXX-XXXXX</p>
+          <p>Remote Yo-Yo Code:</p>
+          <p id ="remote-scad-code">XXXX-XXXXX</p>
```
Comment (mikevanis) [inline data/index.html]: Ok, so this should be:
```
<p id="remote-scad-text">Remote Yo-Yo Code:<br/><span id="remote-scad-code"></span></p>
```

**Comment by andysheen** [comment]
updated requests

**Pull request closed without merging**